certified-connectors/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2149 @@
+              },
+              "required": ["apikey", "companyname"]
+            }
+          }
+        ]
+      }
+    },
+    "/docupass/validate": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "success": {
+                  "type": "boolean",
+                  "description": "success"
+                },
+                "reference": {
+                  "type": "string",
+                  "description": "reference"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "error": {
+                  "type": "object",
+                  "properties": {
+                    "message": {
+                      "type": "string",
+                      "description": "message"
+                    },
+                    "code": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "code"
+                    }
+                  },
+                  "description": "error"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Validating Callback",
+        "description": "To make sure the data was sent from our server.",
+        "operationId": "ValidatingCallback",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "apikey": {
+                  "type": "string",
+                  "description": "Your private API key is available under your web portal.",
+                  "title": "apikey"
+                },
+                "reference": {
+                  "type": "string",
+                  "description": "Reference code received in POST BODY.",
+                  "title": "reference"
+                },
+                "hash": {
+                  "type": "string",
+                  "description": "Hash received in POST BODY.",
+                  "title": "hash"
+                }
+              },
+              "required": ["apikey", "hash", "reference"]
+            }
+          }
+        ]
+      }
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.idanalyzer.com/"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.idanalyzer.com/privacy-policy.html"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "IT Operations;Productivity"
+    }
+  ],
+  "definitions": {},
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {
+    "API \u91d1\u9470": {

```

Comment:
@ccyy915 Can you please remove all the unicodes from all the connector artifacts as they may not be supported in all the Power Automate pages.